### PR TITLE
Linkedin Profile Update for issue #1092

### LINKED ISSUE #1092: [Community] Member profile LinkedIn update
**Current Behavior**
[Layer5-ng community member](https://layer5ng.netlify.app/community/members) profiles now have the ability to include a link to their LinkedIn profile.

**Desired Situation**
Update the @Aisuko profile to include the following front matter:

```
linkedin: bowen-li-10101197
```

- [Source code](https://github.com/layer5io/layer5/tree/layer5-ng/src/collections/members) of member profiles.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by ashutosh1401**
**Description**
The Linkedin Profile has been added.
This PR fixes #1092 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
Good work, @ashutosh1401. 👍 

Thank you for jumping in here.

File: =1.8.0 [hunk 1]
```diff
@@ -0,0 +1,8 @@
++ jquery@3.5.1
```
Comment (Tanuj22) [inline =1.8.0:1]: Remove this file

File: =3.7.0-dev [hunk 2]
```diff
@@ -0,0 +1,8 @@
++ typescript@4.0.5
```
Comment (Tanuj22) [inline =3.7.0-dev:1]: Remove this file

File: package.json [hunk 3]
```diff
@@ -21,9 +21,12 @@
     "preload-fonts": "gatsby-preload-fonts"
   },
   "dependencies": {
+    "@babel/core": "^7.12.3",
```
Comment (Tanuj22) [inline package.json:24]: Revert these changes in dependencies.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (ashutosh1401) [reply package.json:24]: I have to uninstall `@babel/core` for this one

File: package-lock.json [hunk 4]
```diff
@@ -97,34 +97,144 @@
       }
     },
     "@babel/core": {
-      "version": "7.10.4",
-      "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.10.4.tgz",
-      "integrity": "sha512-3A0tS0HWpy4XujGc7QtOIHTeNwUgWaZc/WuS5YQrfhU67jnVmsD6OGPc1AKHH0LJHQICGncy3+YUjIhVlfDdcA==",
+      "version": "7.12.3",
```
Comment (Aisuko) [inline package-lock.json:100]: I believe we should not upgrade the dependency version. Maybe you need to revert the changes on `package-lock.json` and just remove the  package may not enough.

**Review by Aisuko — CHANGES_REQUESTED** [review]
